src/styling/themes.ts

```diff
@@ -11,6 +11,7 @@ export const lightTheme = {
 		// These colors overrides default values from react navigation themes.
 		//  They will influence the default look of navigation-related primitives.
 		text: '#333', // The text color of various elements.
+		textGrey: '#828282', // The text color of various elements.
```

Comment:
textSecondary may be? It is better to use semantic naming for colours in themes, otherwise you will inevitably end up having `textGrayOne`, `textGrayTwo`, `textDarkestGray` etc